examples/lit-icd-app/lit-icd-common/lit-icd-server-app.zap

```diff
@@ -1771,6 +1771,22 @@
               "maxInterval": 65344,
               "reportableChange": 0
             },
+            {
```

Comment:
I think I would expect to see a src/app/zap-templates/zcl/data-model/chip/general-diagnostics-cluster.xml change as well to have a globalAttribute 

Or ar we looking for parallel support of different versions? That may be hard.

Reply:
Reason is: see changes in https://github.com/project-chip/connectedhomeip/pull/30626 where we will start reporting what revisions we have in our xml and codegen. I believe those will be taken from xml rather than zap since zap is per endpoint and cluster revisions are not per endpoint.

Reply:
@cecille XML provides a default value, effectively.  The ZAP UI can then override.

Reply:
So right now, I have it in the xml, but it's also in the zap file because that attribute is turned on. Is it better to turn off the attribute in zap? Or do I just delete the value?

Reply:
ah, ok. That makes sense. Thanks.